test/e2e/framework/network/utils.go

```diff
@@ -94,24 +94,50 @@ var NetexecImageName = imageutils.GetE2EImage(imageutils.Agnhost)
 // Use functional options to configure the NetworkingTest object
 type Option func(*NetworkingTestConfig)
 
+// Endpoints pods listen on SCTP ports
 func EnableSCTP() Option {
 	return func(config *NetworkingTestConfig) {
 		config.SCTPEnabled = true
 	}
 }
 
+// Run the test container with HostNetwork=true.
 func UseHostNetwork() Option {
 	return func(config *NetworkingTestConfig) {
 		config.HostNetwork = true
 	}
 }
 
+// Set the endpoint HTTP port for the test
+func PodHTTPPort(port int) Option {
```

Comment:
you introduce these here but you don't actually use them yet...

Reply:
was meant to simplify the review process only, each commit is not independent